card/src/main/java/com/adyen/checkout/card/StoredCardDelegate.kt

```diff
@@ -51,7 +51,7 @@ import kotlinx.coroutines.flow.onEach
 import kotlinx.coroutines.flow.receiveAsFlow
 import kotlinx.coroutines.launch
 
-@Suppress("TooManyFunctions", "LongParameterList")
+@Suppress("TooManyFunctions", "LongParameterList", "EmptyFunctionBlock")
```

Comment:
You can put a `// no ops` comment in that empty function block, this way you won't need his this rule suppression and we'll know that it's empty on purpose.